### PR TITLE
Add plotting script

### PR DESCRIPTION
**Pull request by IAlibay**
## Input submission checklist

* [ ] created a directory for each system based on the [template directory][templates]
* [ ] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [ ] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [ ] added a PDB file with the protein named `protein.pdb`
* [ ] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [ ] did not re-protonate the residues, but left protonation state of the protein unchanged
* [ ] kept cystallographic waters and metals in the PDB file
* [ ] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [ ] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/utils/extras/plot_results.py [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    Fetch the calculated data from an OpenFE ddG TSV file.
+    
+    Parameters
+    ----------
+    filename : pathlib.Path
+      Pathlib object for the file with the calculated data.
+    
+    Returns
+    -------
+    experimental_data : dict[str, dict[str, float]]
```
Comment (hannahbaumann) [inline industry_benchmarks/utils/extras/plot_results.py:65]:
```suggestion
    calculated_data : dict[str, dict[str, float]]
```

File: industry_benchmarks/utils/extras/plot_results.py [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+    with open(filename, 'r') as fd:
+        rd = csv.reader(fd, delimiter="\t", quotechar='"')
+        headers = next(rd)
+        for row in rd:
+            tag = row[0] + "->" + row[1]
+            calculated_data[tag] = {}
+            calculated_data[tag]['ligand_i'] = row[0]
+            calculated_data[tag]['ligand_j'] = row[1]
+            calculated_data[tag]['dG'] = float(row[2])
```
Comment (hannahbaumann) [inline industry_benchmarks/utils/extras/plot_results.py:79]: Would these be 'DDG'?

File: industry_benchmarks/utils/extras/plot_results.py [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+    fep_err = []
+    for entry in exp_data:
+        exp.append(exp_data[entry]['exp_dG'])
+        exp_err.append(exp_data[entry]['exp_dG_err'])
+        fep.append(exp_data[entry]['fep_dG'])
+        fep_err.append(exp_data[entry]['fep_dG_err'])
+    
+    shift = sum([i['exp_dG'] for i in exp_data.values()]) / len(exp_data)
+    
+    cinnabar_plotting._master_plot(
```
Comment (hannahbaumann) [inline industry_benchmarks/utils/extras/plot_results.py:209]: For some reason the DG Schrodinger vs. Exp. plot says 'DDG' on the axes, do you see the same? Could adding a `plot_type='dg'` or something like that help?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]

File: industry_benchmarks/utils/extras/plot_results.py (hunk 3 above)
Comment (IAlibay) [reply industry_benchmarks/utils/extras/plot_results.py:209]: looks like `quantity` is what was missing, thanks!

File: industry_benchmarks/utils/extras/plot_results.py (hunk 2 above)
Comment (IAlibay) [reply industry_benchmarks/utils/extras/plot_results.py]: nice catch, done!

**Review by hannahbaumann — APPROVED** [review]
Thank you, lgtm! Only thing I noticed that we haven't done a cinnabar release in a while and the change for making the dots more visible is not in yet, but since this is just for internal use that should be ok.

**Comment by IAlibay** [comment]
> Thank you, lgtm! Only thing I noticed that we haven't done a cinnabar release in a while and the change for making the dots more visible is not in yet, but since this is just for internal use that should be ok.

Ah, I was wondering why I didn't see them :(

Yeah, let's make sure we got that fixed for the actual paper.

**Pull request merged**